UnityProject/Assets/Scripts/Weapons/FireLemon.cs

```diff
@@ -0,0 +1,221 @@
+using System.Collections;
+using System.Collections.Generic;
+using System.Linq;
+using Light2D;
+using UnityEngine;
+using UnityEngine.EventSystems;
+using Mirror;
+
+/// <summary>
+/// Generic grenade base.
+/// </summary>
+[RequireComponent(typeof(Pickupable))]
+public class FireLemon : NetworkBehaviour, IPredictedInteractable<HandActivate>, IServerDespawn
+{
+	[Tooltip("Lowest potency explosion")]
+	public Explosion explosionPrefab1;
+
+	[Tooltip("Second lowest potency explosion")]
+	public Explosion explosionPrefab2;
+
+	[Tooltip("Medium potency explosion")]
+	public Explosion explosionPrefab3;
+
+	[Tooltip("Second highest potency explosion")]
+	public Explosion explosionPrefab4;
+
+	[Tooltip("Highest potency explosion")]
+	public Explosion explosionPrefab5;
+
+	[TooltipAttribute("If the fuse is precise or has a degree of error equal to fuselength / 4")]
+	public bool unstableFuse = false;
```

Comment:
Also, please fix case in this script so it would follow up naming conventions of project and wouldn't confuse anyone working with this scripts later - https://unitystation.github.io/unitystation/contribution-guides/Development-Standards-Guide/